src/applications/appeals/10182/containers/FormApp.jsx

```diff
@@ -8,23 +8,23 @@ import { setData } from 'platform/forms-system/src/js/actions';
 
 import formConfig from '../config/form';
 import {
-  noticeOfDisagreementFeature,
+  nodPart3UpdateFeature,
   issuesNeedUpdating,
   getSelected,
   getIssueNameAndDate,
   processContestableIssues,
 } from '../utils/helpers';
 
-import { copyAreaOfDisagreementOptions } from '../utils/disagreement';
+import { SHOW_PART3 } from '../constants';
 
-import { showWorkInProgress } from '../content/WorkInProgressMessage';
+import { copyAreaOfDisagreementOptions } from '../utils/disagreement';
 
 import { getContestableIssues as getContestableIssuesAction } from '../actions';
 
 export const FormApp = ({
   isLoading,
   loggedIn,
-  showNod,
```

Comment:
`showNod` was used when we released NOD using the feature flag. We've been at 100% for quite a while now, so we really don't need the feature flag any more. Eventually we'll remove it completely from the app. I figure that it would hurt to reduce the code.